es/authentication_authorization/README.md

```diff
@@ -34,18 +34,18 @@ No te olvides del decorador encima de `post_edit`, `post_remove`, `post_draft_li
 
 Ahora podemos intentar hacer muchas cosas mágicas para implementar usuarios y contraseñas y autenticación, pero hacer esto correctamente es complicado. Como django viene con "baterías incluidas", alguien ya ha hecho el trabajo duro por nosotros, así que vamos a utilizarlas.
 
-En `mysite/urls.py` agrega una url `url(r'^accounts/login/$', views.login, name='login')`. Así el archivo debería verse similar a este:
+En `mysite/urls.py` agrega una url `path('accounts/login/', views.login, name='login')`. Así el archivo debería verse similar a este:
 
 ```python
-from django.conf.urls import include, url
+from django.urls import include, path
```

Comment:
The English version has `path, include` instead of `include, path`. If there isn't any reason to have it different for Spanish, I'd recommend to adopt the order from the English version for easier future maintaining.

Reply:
It should probably be changed in English so that import are sorted alphabetically?

Reply:
> It should probably be changed in English so that import are sorted alphabetically?

Is there any accepted conventions that says they should be alphabetical? https://pep8.org/#imports doesn't seem to mention that.

Reply:
It does not, but it does make it easier to read/find as it is the "natural" order one expects.

Reply:
I think the current order comes from how the imports are added: [Importing `path` is already part](https://tutorial.djangogirls.org/en/django_urls/#how-do-urls-work-in-django) of what `django-admin startproject mysite .` generated in chapter [Your first Django project!](https://tutorial.djangogirls.org/en/django_start_project/) of the main Django Girls tutorial. The import of `include` is added after that in [Django URLs > Your first Django URL!](https://tutorial.djangogirls.org/en/django_urls/#your-first-django-url).

So if we want to change this, we should also change it everywhere in the main tutorial. But I'd vote not to, as it makes the necessary change in [Django URLs > Your first Django URL!](https://tutorial.djangogirls.org/en/django_urls/#your-first-django-url) harder to spot, when the line still starts and ends as before, and just something in the middle has to be added. Already with the current instructions, where the end of the line changes (due to addition of `, include`), many workshop participants tend to miss that and get an error message later on.

If you'd like to have it changed, anyway, @goanpeca, please file an issue and/or pull request on the repo of the main tutorial (https://github.com/DjangoGirls/tutorial) and we can discuss further there.

Reply:
You raise a good point! I will open an issue for further discussion, but I agree it is best not to change this based on the original content :-)